client/src/components/Navbar.js

```diff
@@ -39,43 +33,127 @@ const NavbarContainer = styled.div`
     ${props => {
       if(props.isColored){
         return (
-          'background: rgba(81, 39, 255, 0.9); \n transition: background 0.3s;'
+          `background: ${props.theme.color.background_1};
+           transition: background 0.3s;`
           )
         };
       }
     };
   };
 `;
+
+const MainAligner = styled.div`
+ width: 100%;
+ display: flex;
+ justify-content: space-between;
+`;
+
+const NavLI = styled.li`
```

Comment:
I chose this name because `NavbarItem` is already taken and this element only refers to the List Items as opposed to links redirecting to external URLs